js/controller.js

```diff
@@ -749,11 +750,6 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
           this.state.screenToShow = CONSTANTS.SCREEN.PAUSE_SCREEN;
         }
         this.state.playerState = CONSTANTS.STATE.PAUSE;
-
-        // [PLAYER-2220]: videoVr should not blur. This prevents a circular review on a pause.
-        if (!this.videoVr) {
-          this.state.mainVideoElement.classList.add('oo-blur');
```

Comment:
It looks like this used to be executed even if the condition on the if statement where `addBlur()` is called was false. Will this change any behavior?

Reply:
Ok, sounds good